.github/workflows/smoke.yml

```diff
@@ -35,8 +39,16 @@ jobs:
       uses: actions/checkout@v3
 
     - name: Setup LXD
+      if: matrix.cloud == 'localhost'
       uses: canonical/setup-lxd@90d76101915da56a42a562ba766b1a77019242fd
 
+    - name: Setup MicroK8s
+      if: matrix.cloud == 'microk8s'
+      uses: balchua/microk8s-actions@v0.3.1
+      with:
+        channel: "1.23/stable"
```

Comment:
We can install `1.25/stable`, and enable `dns`, `hostpath-storage`(storage is deprecated), and `RBAC`(then we can test RBAC).